packages/expo-image/src/ExpoImage.web.tsx

```diff
@@ -51,7 +52,10 @@ export default function ExpoImage({
   ...props
 }: ImageNativeProps) {
   const { aspectRatio, backgroundColor, transform, borderColor, ...style } = props.style ?? {};
-
+  const imagePlaceholderContentFit = placeholderContentFit || 'scale-down';
+  const blurhashStyle = {
+    objectFit: placeholderContentFit || contentFit,
```

Comment:
On mobile platforms, blurhashes always use `contentFit`, but perhaps we should discuss it and do as you did here. cc @lukmccall

Reply:
hmm, If you set `placeholderContentFit` you probably expect it to apply to all placeholders, the different behaviour of blurhash and image placeholders was just a smart default

Reply:
I don't have a strong opinion, but setting it to the `placeholderContentFit` makes more sense if you ask me.